app/models/user.rb

```diff
@@ -13,32 +13,20 @@ class User < ApplicationRecord
   # @citizen428 Preparing to drop profile columns from the users table
   PROFILE_COLUMNS = %w[
     available_for
-    behance_url
```

Comment:
This is used for `self.ignored_columns` below, don't we want to keep on ignoring them? There's little point in deserializing them from the DB if we don't use them in the app.